.circleci/config.yml

```
@@ -88,39 +88,18 @@ jobs:
 
   style:
     docker:
-      - image: circleci/buildpack-deps:trusty
+      - image: nixos/nix:2.3
     steps:
       - checkout
-      - restore_cache:
-          keys:
-          - v1-stack-style-dependencies-{{ checksum "stack.yaml" }}
-      - run:
-          name: install stack
-          command: |
-            curl -L https://github.com/commercialhaskell/stack/releases/download/v2.1.3/stack-2.1.3-linux-x86_64.tar.gz | tar zx -C /tmp
-            sudo mv /tmp/stack-2.1.3-linux-x86_64/stack /usr/bin
-            stack setup
       - run:
-          name: install linters
-          command: |
-            # -fno-spec-constr may help with out-of-memory issues compiling
-            # haskell-src-exts. Compare stack.yaml.
-            stack install --ghc-options=-fno-spec-constr hlint stylish-haskell
-      - save_cache:
-          paths:
-            - "~/.stack"
-            - ".stack-work"
-          key: v1-stack-style-dependencies-{{ checksum "stack.yaml" }}
-      - run:
-          name: Add stack tools to $PATH
-          command: |
-            echo "export PATH=/home/circleci/.local/bin:$PATH" >> $BASH_ENV
+          name: Install linting and styling scripts
+          command: nix-env -f default.nix -iA lint style
       - run:
-          name: run linter
-          command: make lint
+          name: Run linter
+          command: postgrest-lint
```

For non-nix users, do you think we could suggest here a way of checking the failure locally?

I'm thinking of having an `echo` like:

```suggestion
          name: Run linter
          command: |
            echo "Note: For checking this locally, use `make lint`. Or, if using `nix`, run `nix-shell --run postgrest-lint`.
            postgrest-lint
```